cmd/openfga/openfga.go

```diff
@@ -13,6 +13,6 @@ func main() {
 	rootCmd.AddCommand(runCmd)
 
 	if err := rootCmd.Execute(); err != nil {
-		log.Fatal(err)
+		os.Exit(1)
```

Comment:
os.Exit here instead because `Execute` already prints out errors, so the `log.Fatal` is redundant.